TESTS/mbed_platform/minimal-printf/compliance/test_app.json

```diff
@@ -0,0 +1,7 @@
+{
+    "target_overrides": {
+        "*": {
+            "platform.minimal-printf-enable-floating-point": true
```

Comment:
You will need to pass the option when running greentea tests
--app-config ./TESTS/minimal-printf/compliance/test_app.json

Reply:
We will instead define the macro in the `main.cpp` of the test.